iwyu_globals.cc

```diff
@@ -296,16 +296,14 @@ static vector<HeaderSearchPath> NormalizeHeaderSearchPaths(
 static vector<HeaderSearchPath> ComputeHeaderSearchPaths(
     clang::HeaderSearch* header_search) {
   map<string, HeaderSearchPath::Type> search_path_map;
-  for (clang::HeaderSearch::search_dir_iterator
-           it = header_search->system_dir_begin();
+  for (auto it = header_search->system_dir_begin();
```

Comment:
Here I'm on the fence regarding change to `auto`. On one hand iterator type is not obvious, on the other hand `clang::HeaderSearch::search_dir_iterator` isn't significantly more useful.